packages/server/api/src/app/webhooks/webhook-controller.ts

```diff
@@ -52,6 +61,8 @@ export const webhookController: FastifyPluginAsyncTypebox = async (app) => {
                 ),
                 flowVersionToRun: WebhookFlowVersionToRun.LOCKED_FALL_BACK_TO_LATEST,
                 execute: true,
+                parentRunId,
```

Comment:
I would rename it to `...extractOptionsFromRequest(request)` and just make it return the both flag and use it in all functions.